### ISSUE TITLE
Create specification docs for the Scheduler

### ISSUE THREAD

**Issue by MuellMark** [MuellMark/Course-Scheduler#7]
I've worked on this for about half an hour so far. While I know all of the considerations and specifications that go into the scheduler, I can't assume anyone else does. For this reason, I'm going to go ahead and create some google docs outlining what constraints need to be considered by the scheduler to keep in mind.

So far I've made a general specifications doc and a scheduler specifications, the general one is made to have things to keep in mind and other pointers, while the scheduler one has specific considerations for the constrains in the linear program. These are mostly used in the python, but it's still a good resource for us to have

**Comment by MuellMark** [comment]
I've added all of the schedule restrictions to the document that were outlined for me by Dr. Anthony when I started the research, expect I also gave explanations to how they are used in the CSV Files. 

I'm now working on a document that explains my course abbreviation scheme, which got me thinking on how we should incorporate them into the site. Should these abbreviations be communicated to the users? Or should the users create their own and specify what the course name is? I'll think on this further as I wrap up what I'm doing this week

**Comment by MuellMark** [comment]
I'm going to wrap up this issue in the next half hour after I finalize the documents and find the abbreviations I made. The link to it is right here: https://docs.google.com/document/d/1l72yqoqTNTOHYHCQkiu0IRupf_f5YN1LTWGkdNcFy9w/edit

**Comment by MuellMark** [comment]
I thought I'd be done with this issue but I have one more thing I'm creating that will help with #9. The CSV's need to have a specific lay out to them, which I'll describe in 2 docs. This is especially important because I'm anticipating some changes to them that I'll outline in a separate issue.

**Comment by MuellMark** [comment]
I've completed (at least, for the time being) a series of 5 files that describe that give more guidance on the existing scheduler code. I have plans on altering the CSV's to add extra features for the site, but this will be its own separate issue.

Finalizing everything didn't take a full half an hour so I set up my branch, as I'm also going to look into being able to call a python script from within an html page over the next few days. My commit is referencing this, and was a test of GitKraken (as per Colby's request). 

This issue can be closed with the pretext that any alterations to the docs (which I am anticipating) can be noted in future threads
